### DISCUSSION TITLE
Welcome to shopify-vite Discussions!

### DISCUSSION THREAD

**Discussion by montalvomiguelo** [barrel/shopify-vite#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrJulik** [comment]
Hello everyone. My name is Yuliy and I'm a full stack developer at BrickspaceLab. We have recently moved our Shopify themes workflow from webpack to vite, all thanks to your awesome plugin. We have also [released a toolkit](https://github.com/BrickspaceLab/paper-toolkit) to simplify working with our Paper theme. Looking forward to contributing to the project!

**Comment by knjshimi** [comment]
Hey guys, I’m Kenji and I work at The Working Party in AU. I’m trying to move away from webpack to Vite, and yeah, vite-plugin-shopify is how I’m trying.

I believe the last piece of the puzzle is actually not really related to shopify, but rather making prefresh (preact) to work with this workflow. I’m keen to know if you guys have HMR working nicely with this plugin and vite!

**Reply by montalvomiguelo** [reply]
Hey @kenji-twp, I use Preact in my projects at Barrel, and HMR works nicely. I use this plugin [`@preact/preset-vite`](https://www.npmjs.com/package/@preact/preset-vite) (:

**Reply by knjshimi** [reply]
Thanks @montalvomiguelo! I'm having some issues with both solutions (prefresh standalone and preset-preact), so maybe it's related to how our preact code has been structured. I'll have a look.

But anyway, excellent work with this plugin. I'm keen to help if I can ☺️

**Reply by montalvomiguelo** [reply]
@kenji-twp lets try adding our `frontend/` folder to the [`.shopifyignore`](https://github.com/barrel/shopify-vite/blob/main/examples/react-shopify/.shopifyignore#L3) file so that the Shopify CLI does not refresh the page when we change our JSX/TSX files there.

**Comment by andyxmas** [comment]
Hi all, my name is Andy Christmas, I'm the lead front end developer at Pixelcabin. We've used vite-plugin-shopify in the last few client projects we've worked on. Very grateful for all your excellent work on this @montalvomiguelo!
